src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/MinMax_DT.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    // Min(arg1:n, arg2:n, ..., argN:n)
+    // Max(arg1:n, arg2:n, ..., argN:n)
+    // Corresponding Excel functions: Min, Max
+    internal sealed class MinMaxFunction_DT : BuiltinFunction
+    {
+        public override bool IsSelfContained => true;
+
+        private readonly bool _isMin;
+
+        public override string GetUniqueTexlRuntimeName(bool isPrefetching = false)
+        {
+            if (_isMin)
+            {
+                return BuiltinFunctionsCore.Min.GetUniqueTexlRuntimeName(isPrefetching);
+            }
+            else
+            {
+                return BuiltinFunctionsCore.Max.GetUniqueTexlRuntimeName(isPrefetching);
+            }
+        }
+
+        public MinMaxFunction_DT(bool isMin)
+            : base(isMin ? "Min" : "Max", isMin ? TexlStrings.AboutMin : TexlStrings.AboutMax, FunctionCategories.MathAndStat, DType.DateTime, 0, 1, int.MaxValue, DType.DateTime)
```

Comment:
The return type won't necessarily be DType.DateTime (it can be either Time or Date, depending on the arguments)